### ISSUE TITLE
Installation error 

### ISSUE THREAD

**Issue by saimaafrin** [marius-team/marius#141]
I am getting the error below while installing marius. I have tried several solutions but those didn't work. 

''' ERROR: Failed building wheel for marius
Failed to build marius
ERROR: Could not build wheels for marius, which is required to install pyproject.toml-based projects '''
what could be the issue of this error?

**Comment by rogerwaleffe** [comment]
Thanks for the question and sorry for the delayed response. We have been in the process of updating dependency versions and the build and install for Marius/MariusGNN.

Can you try the installation again using the instructions here: https://github.com/marius-team/marius/tree/main/examples/docker

**Comment by saimaafrin** [comment]
thanks for the reply. the docker installation did work for me but the direct installation failed even after fulfilling the mentioned requirements. you can see below: 
Requirements ------->   (I have)

* CUDA >= 10.1 ——> 11.3
* CuDNN >= 7 ——> 8
* pytorch >= 1.8 (pip3 install torch==1.11.0+cu113 torchvision==0.12.0+cu113 torchaudio==0.11.0 --extra-index-url https://download.pytorch.org/whl/cu113) —> 1.11.0+cu113
* python >= 3.6 —> Python 3.8.10
* GCC >= 7 (On Linux) or Clang 12.0 (On MacOS) —> gcc (Ubuntu 8.4.0-3ubuntu2) 8.4.0
* cmake >= 3.12 ——> cmake version 3.26.4
* make >= 3.8 ——> GNU Make 4.2.1

**Comment by rogerwaleffe** [comment]
Yeah it's possible that some combinations of the above requirements and environments may not work. It's hard for us to test every possible configuration, hence we try to provide a few exact environments that we know work (e.g., with docker).

For your specific case, can you provide us with a bit more information on the error you are getting? It would be helpful to see the full trace if possible and more details about your exact environment.

You can run `marius_env_info` or `python3 src/python/distribution/marius_env_info.py` to get information about your system.

You can also try running the following (rather than `pip3 install .`) if you wish to try to build/investigate the error manually:
```
mkdir build
cd build
cmake ../ -DUSE_CUDA=1 -DUSE_OMP=1 
make marius_train -j
make bindings -j
cd ..
```

**Comment by yxr620** [comment]
I ran into this problem too and I was using the latest main branch. the error looks like this:
```
Requirement already satisfied: mpmath>=0.19 in /usr/local/lib/python3.10/dist-packages (from sympy->torch>=1.7.1->marius==0.0.2) (1.3.0)
Building wheels for collected packages: marius
  Building wheel for marius (pyproject.toml) ... error
  error: subprocess-exited-with-error
  
  × Building wheel for marius (pyproject.toml) did not run successfully.
  │ exit code: 1
  ╰─> [99 lines of output]
      running bdist_wheel
      running build
      running build_py
      running egg_info
      writing marius.egg-info/PKG-INFO
      
     .......
     
           subprocess.CalledProcessError: Command '['cmake', '/working_dir', '-DCMAKE_LIBRARY_OUTPUT_DIRECTORY=/working_dir/build/lib.linux-x86_64-3.10/marius', '-DPYTHON_EXECUTABLE=/usr/bin/python3', '-DCMAKE_BUILD_TYPE=Release', '-DCMAKE_BUILD_WITH_INSTALL_RPATH=TRUE', '-DCMAKE_INSTALL_RPATH_USE_LINK_PATH=TRUE', '-DUSE_CUDA=TRUE', '-DUSE_OMP=TRUE', '-DCMAKE_INSTALL_RPATH=$ORIGIN']' returned non-zero exit status 1.
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for marius
Failed to build marius
ERROR: Could not build wheels for marius, which is required to install pyproject.toml-based projects
```

**Comment by hulihan-start** [comment]
> Yeah it's possible that some combinations of the above requirements and environments may not work. It's hard for us to test every possible configuration, hence we try to provide a few exact environments that we know work (e.g., with docker).
> 
> For your specific case, can you provide us with a bit more information on the error you are getting? It would be helpful to see the full trace if possible and more details about your exact environment.
> 
> You can run `marius_env_info` or `python3 src/python/distribution/marius_env_info.py` to get information about your system.
> 
> You can also try running the following (rather than `pip3 install .`) if you wish to try to build/investigate the error manually:
> 
> ```
> mkdir build
> cd build
> cmake ../ -DUSE_CUDA=1 -DUSE_OMP=1 
> make marius_train -j
> make bindings -j
> cd ..
> ```

Hi, I successfully install marius by this method, but when I try to use marius_train, it returns: bash: marius_train: command not found

And I try to use python3 examples/python/fb15k_237_gpu.py, then it returns ModuleNotFoundError: No module named 'marius'

Is there any suggestion for solving this issue? Thank you!
